gratipay/models/participant/routes.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import braintree
+from psycopg2 import IntegrityError, errorcodes
+
+from ..exchange_route import ExchangeRoute
+from ...billing.instruments import CreditCard
+from ...utils import is_card_expiring
+
+
+class Routes(object):
```

Comment:
Would it make more sense to call this `PaymentRoutes` since you also have the concept of routes wiring HTTP requests to code that handles them?